app/controllers/content_item_signups_controller.rb

```diff
@@ -26,7 +26,7 @@ def confirm
   def create
     signup = ContentItemSubscriberList.new(content_item.to_h)
```

Comment:
Could we move this within the if statement too, so we're only doing this once we know it's present?

Reply:
Oh yes, good point - updated.